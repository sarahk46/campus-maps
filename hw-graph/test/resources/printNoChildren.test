# Tests that a graph can be created along with one node, and
# no edge can be printed out.

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Print the outgoing edges from n1
ListChildren graph1 n1