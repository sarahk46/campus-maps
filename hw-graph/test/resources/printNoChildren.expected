# Tests that a graph can be created along with one node, and
# no edge can be printed out.

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Print the outgoing edges from n1
the children of n1 in graph1 are: